docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>Maybe instead of "3 leaves", "5 leaves", write precisely what the leaves are, e.g. "3 leaves: 1, (2, 3), ()"; "5 leaves: 1, 2, (3, 4), 5"</p><p> </p><p>Oh wait, I think the second example is wrong, doesn't it have 4 leaves? (2 and 3 are distinct?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Unless I didn't understand something, the second example is correct: leaves are 1,2,3 as 2, 3 are packed in a tuple which is also a container.</p>